pkgs/servers/caddy/default.nix

```diff
@@ -26,11 +23,4 @@ buildGoModule rec {
     }
     EOF
   '';
-
-  meta = with stdenv.lib; {
-    homepage = "https://caddyserver.com";
-    description = "Fast, cross-platform HTTP/2 web server with automatic HTTPS";
-    license = licenses.asl20;
-    maintainers = with maintainers; [ rushmorem fpletz zimbatm filalex77 ];
-  };
-}
+})
```

Comment:
same goes for the meta information. In fact, meta information might actually be different.